quotes.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+	<html>
+	<head>
+		<link rel="stylesheet" type="text/css" href="css/styles.css">
+		<title>KANYE RESUME</title>
+	</head>
+
+	<body>
+		<nav></nav>
+		<nav></nav>
+			
+			<div class="header">
+				<h1>KANYE WEST</h1>
+
+				<ul>
+					<li style="color: black">QUOTES</li>
+					<li style="color: black"><b>||</b></li>
+					<li>
+						<a href="index.html">ME, MYSELF & I
```

Comment:
:hammer: Forget to close something? Seems like everything on the page is one big link now...